### ISSUE TITLE
Use only Logo from Theming in emails without background color

### ISSUE THREAD

**Issue by reos-rcrozier** [nextcloud/server#31320]
### Is your feature request related to a problem? Please describe.

Currently if you modify the theming in Nextcloud Settings->Theming calendar emails are modified also, however, the modification is very crude. For example in my theming I have a dark green background color, and the themed emal has a dark green blob at the top overlaid with black text containing the contents of the 'Name' field from the Theming setting. This is hard to read and looks very unprofessional.
 

### Describe the solution you'd like

If the user has provided a logo in the theming section, display this logo at the top of calendar emails instead of the colored blob with overlaid text. If you don't have a logo at least choose a suitable text color like white if a dark background color has been selected.

**Comment by acsfer** [comment]
What about if you use a full-white logo in your Nextcloud? Without a background, logo won't display.

**Comment by reos-rcrozier** [comment]
@acsfer, I'm not sure exactly what you mean by a full-white logo? Or exactly what you mean by "Without a background, logo won't display". 

I have tried a white logo foreground with transparent background (but the logo shown above has transparent background anyway). This doesn't make a difference to the issue that before logos are loaded the replacement text in the email is black, which doesn't look so good on a dark background. 

Basically it would be nice to expose some aspects of email themes in the the Theming interface. For example the ability to change the header background color separately from the Nextcloud theme color, and the ability to change the alternate text color for the logo, and perhaps even choose a different logo file for the emails.

**Comment by acsfer** [comment]
Use this logo. https://logoseducation.group/wp-content/uploads/2020/10/White-logo-no-background.png

And imagine it on the mail body without a color background (withe logo + white background).

> Basically it would be nice to expose some aspects of email themes in the the Theming interface. For example the ability to change the header background color separately from the Nextcloud theme color, and the ability to change the alternate text color for the logo, and perhaps even choose a different logo file for the emails.

This can be done by overriding the emails class (not by the WebUI at this time).

**Comment by papahausa** [comment]
hello. i have the same. where in mail templates i can remove this backgroung? 
![Screenshot_5](https://github.com/nextcloud/server/assets/106929039/3bbbcb0f-31db-457b-89e3-8d70a87fe420)

**Comment by skjnldsv** [comment]
Hi,

I'll close this ticket. We don't allow customizing emails templates to that extreme. Otherwise we'll never stop oroviding configs switches or customization options :(
There is some discussion regarding a new email design here: https://github.com/nextcloud/server/issues/48095
Thank you :)
